react/features/welcome/reducer.js

```diff
@@ -1,19 +1,41 @@
 // @flow
 
 import { ReducerRegistry } from '../base/redux';
-import { SET_SIDEBAR_VISIBLE } from './actionTypes';
+import { PersistenceRegistry } from '../base/storage';
```

Comment:
All of the above are imports from another feature and the one bellow is an import from the same feature. According to our coding style, the group must be separated by a line.